custom_components/localtuya/common.py

```diff
@@ -260,6 +260,14 @@ def dps(self, dps_index):
 
         return value
 
+    def dps_conf(self, conf_item):
+        """Return value of datapoint for user speified config item.
```

Comment:
Typo: specified. Wondering why it has not been detected by tox ?

Reply:
Ah, right. I think Code spell has a dictionary of commonly misspelled words, so it's not foolproof. Since it spell checks variable and function names too, there has to be a balance to what to complain at otherwise a lot of common (and domain specific) words would be treated as errors. Since you can refer to variable names in comments, you can't really make exceptions based on that.